web/docs/language/overview.md

```diff
@@ -7,11 +7,13 @@ words (types) of the language.
 
 The basic idea is that the higher-level overview of an app (e.g. pages, routes, database model, ...) is defined in `*.wasp` files (for now just one), while the specific parts (web components, back-end queries, ...) are implemented in specific non-Wasp technologies (React, NodeJS, Prisma) and then referenced in the `*.wasp` files.
 
-Basic structure of a Wasp project is:
+The basic structure of a Wasp project is:
 - `*.wasp` file
-- The `src/` folder -> Contains non-Wasp code (JS, CSS, ...). You can structure it however you want, as long as you put it somewhere inside the correct subfolder:
+- The `src/` folder
+
+The `src/` folder contains non-Wasp code (JS, CSS, ...). You can structure it however you want, as long as you put it somewhere inside the correct subfolder:
   - The `src/server` folder - Contains your server code (i.e., executed by Node JS).
-  - The `src/client` folder - Contains your client code (i.e., executed by JS in user's browsers).
+  - The `src/client` folder - Contains your client code (i.e., used in the browser).
```

Comment:
We want to emphasize where the code is executed (i.e., its runtime), so let's stick with "executed by the user's browser" or similar.

Reply:
I agree that's the more precise way to write it, but I think people will infer from `client` and `it's in the browser` that's the browser runtime, keeping it light for our more beginner users.